Resources/config/console.xml

```diff
@@ -16,6 +16,14 @@
             <argument type="collection" /> <!-- user providers -->
             <tag name="console.command" command="lexik:jwt:generate-token" />
         </service>
+
+        <service id="lexik_jwt_authentication.generate_keypair_command" class="Lexik\Bundle\JWTAuthenticationBundle\Command\GenerateKeyPairCommand">
+            <argument type="service" id="filesystem" />
+            <argument type="abstract" />
+            <argument type="abstract" />
+            <argument type="abstract" />
```

Comment:
you can omit the type here (`abstract` is symfony 5.x only while this bundle supports 4.4)

Reply:
OK, will do.

Another thing to anticipate about the Flex recipe: correct me if I'm wrong it looks like Flex has no option about running a command during the flex recipe installation (only composer post-install, meaning it would run on every `composer install`, and this is not what we want).
One way to address this would be to add a kinda `--first-run` option, which would check for the keys existence, and dismiss if they're already here. Then we would be able to add it safely as a post-install script. WDYT?

Reply:
Another, safer, but less developper-friendly, would just be to add a `post-install.txt` in the flex recipe to suggest the user to run this command, then we don't need that extra behavior.

Reply:
A `--skip-if-exists` option sounds good to me, in addition to a `--overwrite` to force regeneration (which cannot be combined with the first one).

Reply:
Hm, I don't get it. If we have an `--skip-if-exists` option, this involves that the default behavior is overwriting, isn't it? So why having an additional `--overwrite` option?

Reply:
Well, if that makes more sense, I now consider that keys are not supposed to exist or an error is raised, except when using `--skip-if-exists` or `--overwrite` (I think that's what you initially meant).
I added a bunch of new tests about this (plus an encryption/decryption test) and explained that behavior in the documentation.